DEVELOPERS.md

```diff
@@ -0,0 +1,214 @@
+# Developing for Accord Project
+
+* [Development Setup][developers.setup]
+* [Coding Rules][developers.rules]
+* [Commit Message Guidelines][developers.commits]
+* [Writing Documentation][developers.documentation]
+
+## <a name="setup"> Development Setup
+
+This document describes how to set up your development environment to build and test Cicero-UI, and
+explains the basic mechanics of using `git`, `node`, `npm`.
+
+### Installing Dependencies
+
+Before you can build Cicero-UI, you must install and configure the following dependencies on your
+machine:
+
+* [Git][git]: The [Github Guide to Installing Git][git-setup] is a good source of information.
+
+* [Node.js v10.16.0 'Dubnium' (LTS)][node]: We use Node to generate the documentation, run a
+  development web server, run tests, and generate distributable files. Depending on your system,
+  you can install Node either from source or as a pre-packaged bundle.
+
+  We recommend using [nvm][nvm] (or [nvm-windows][nvm-windows])
+  to manage and install Node.js, which makes it easy to change the version of Node.js per project.
+
+### Forking Accord Project repositories on Github
+
+To contribute code to Accord Project, you must have a GitHub account so you can push code to your own
+fork of an Accord Project repository and open Pull Requests in the repository.
+
+To create a Github account, follow the instructions [here][github-signup].
+Afterwards, go ahead and [fork][github-forking] the specific Accord Project repository.
+
+### Building a Project
+
+To build a project, you clone the source code repository and use npm to build:
+
+```shell
+# Clone your Github repository:
+git clone https://github.com/<github username>/<repository>.git
+
+# Go to the directory:
+cd <repository>
+
+# Add the main repository as an upstream remote to your repository:
+git remote add upstream "https://github.com/acccordproject/<repository>.git"
+
+# Install node.js dependencies:
+npm install
+```
+
+### Running the Project Locally
```

Comment:
This does not seem to apply to this project. Please review and update the `DEVELOPERS.md` file so it reflects this project.